web/modules/custom/talkcodemodule/src/Plugin/Block/talkcodemoduleblock.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Drupal\talkcodemodule\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides a 'talkcodemoduleblock' block.
+ *
+ * @Block(
+ *  id = "talkcodemoduleblock",
+ *  admin_label = @Translation("Talkcodemoduleblock"),
+ * )
+ */
+class talkcodemoduleblock extends BlockBase {
+
+
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultConfiguration() {
+    return [
+         'title' => $this->t('BUILD IT BOLD AND BEAUTIFUL'),
+         'subtitle' => $this->t('Work with a reliable developer and build a brilliant digital experience'),
+         'talkcodebutton' => $this->t('LETS TALK CODE'),
+        ] + parent::defaultConfiguration();
+
+ }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function blockForm($form, FormStateInterface $form_state) {
+    $form['title'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Title'),
+      '#description' => $this->t('Talk Code Title'),
+      '#default_value' => $this->configuration['title'],
+      '#maxlength' => 64,
+      '#size' => 64,
+      '#weight' => '0',
+    ];
+    $form['subtitle'] = [
+      '#type' => 'textarea',
+      '#title' => $this->t('Subtitle'),
+      '#description' => $this->t('Talk Code Subtitle'),
+      '#default_value' => $this->configuration['subtitle'],
+      '#weight' => '1',
+    ];
+    $form['talkcodebutton'] = [
+      '#type' => 'textfield',
```

Comment:
This textfield stores the text for the button. You should have another textfield (or URL/link type) for storing the actual link that the button points to.